src/util/defines.hpp

```diff
@@ -88,4 +88,9 @@ decltype(wr::SceneGraph::m_update_lights_func_impl) wr::SceneGraph::m_update_lig
 decltype(wr::SceneGraph::m_update_transforms_func_impl) wr::SceneGraph::m_update_transforms_func_impl = [](wr::RenderSystem* render_system, wr::SceneGraph& scene_graph, std::shared_ptr<wr::Node>& node) \
 { \
 	static_cast<renderer_type*>(render_system)->function(scene_graph, node); \
+};
+#define LINK_SG_DELETE_SKYBOX(renderer_type, function) \
```

Comment:
look at this github highlighting though :o

Reply:
:o